packages/history/src/models/migrations/20200320081228-insert-WorkflowTransHist-indexes.js

```diff
@@ -5,14 +5,20 @@ const FINISHEDON_INDEX = 'WorkflowTransitionHistory_FinishedOn';
 
 module.exports = {
   up: (queryInterface) => {
-    queryInterface.addIndex(TABLE, {
-      fields: ['businessObjId', 'businessObjType', 'to'],
-      name: SEARCH_INDEX
-    });
-    queryInterface.addIndex(TABLE, {
-      fields: ['finishedOn'],
-      name: FINISHEDON_INDEX
-    });
+    queryInterface.addIndex(
+      TABLE,
+      ['businessObjId', 'businessObjType', 'to'],
```

Comment:
Please revert this change, because according to https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html#instance-method-addIndex previous definition is recommended:
![Screenshot 2021-09-22 at 13 25 20](https://user-images.githubusercontent.com/31243790/134327446-ed3ff03c-58a4-4f39-b760-cfd85b3e8292.png)